spec/requests/api/v3/repositories/revisions_resource_spec.rb

```diff
@@ -0,0 +1,129 @@
+#-- copyright
+# OpenProject is a project management system.
+# Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See doc/COPYRIGHT.rdoc for more details.
+#++
+
+require 'spec_helper'
+require 'rack/test'
+
+describe 'API v3 Revisions resource', type: :request do
+  include Rack::Test::Methods
+  include Capybara::RSpecMatchers
+  include API::V3::Utilities::PathHelper
+
+  let(:revision) {
+    FactoryGirl.create(:changeset,
+                       repository: repository,
+                       comments: 'Some commit message',
+                       committer: 'foo bar <foo@example.org>'
+    )
+  }
+  let(:repository) {
+    FactoryGirl.create(:repository_subversion, project: project)
+  }
+  let(:project) {
+    FactoryGirl.create(:project, identifier: 'test_project', is_public: false)
+  }
+  let(:role) {
+    FactoryGirl.create(:role,
+                       permissions: [:view_changesets])
+  }
+  let(:current_user) {
+    FactoryGirl.create(:user, member_in_project: project, member_through_role: role)
+  }
+
+  let(:unauthorized_user) { FactoryGirl.create(:user) }
+
+  describe '#get' do
+    let(:get_path) { api_v3_paths.revision revision.id }
+    let(:expected_response) do
```

Comment:
I've never seen such a detailed expectation. Not sure if this will work everywhere.

But if I am not mistaken, you are using this expected response nowhere.